### PR TITLE
Add dev env

### PR DESCRIPTION
**Pull request by Jnoack331**
I just wrote some scripts to allow easier development through a docker container

### CODE REVIEW

**Review by michz — DISMISSED** [review]

File: scripts/importPublicKey.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+echo "${PUBLIC_KEY}" >> ~/.ssh/authorized_keys
+passwd -d root
+chown root:root /root/.ssh/authorized_keys
+
+apt-get update
+apt-get install -y python
```
Comment (sd-lueckel) [inline scripts/importPublicKey.sh:8]: Why not installing this packages all together instead one-by-one?

**Review by sd-lueckel — CHANGES_REQUESTED** [review]

File: scripts/importPublicKey.sh (hunk 1 above)
Comment (Jnoack331) [reply scripts/importPublicKey.sh:8]: because if one fails all fail and if you are using ubuntu 16.04 gpg is not even a package and in 18.04 its needed

Comment (sd-lueckel) [reply scripts/importPublicKey.sh:8]: Yeah, but they are all needed to run the system or not? Then it is ok if it fails if 1 package fails!

And if there is a difference between 16.04 and 18.04 there should be defined different packages for the system for example through a specific file which will be included based on the detected system

Comment (Jnoack331) [reply scripts/importPublicKey.sh:8]: Im not gonna implement an OS detecting mechanism for an development environment so just install python everything else has the role to install anyway

**Review by sd-lueckel — APPROVED** [review]

**Review by mattagohni — APPROVED** [review]

**Pull request merged**